docs/API/v3/package-base-address-resource.md

```diff
@@ -0,0 +1,122 @@
+---
+# required metadata 
+
+title: Package Content | NuGet V3 API | Microsoft Docs
+author:
+- joelverhagen
+- kraigb
+ms.author:
+- joelverhagen
+- kraigb
+manager: skofman
+ms.date: 10/26/2017
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+ms.assetid: ec68b5d1-a684-4995-b1a6-6210dbb24875
+
+# optional metadata
+
+description: The package base address is a simple interface for fetching the package itself.
+keywords: flat container, dnx, package base address, nupkg, versions, unlisted, enumerate, download
+ms.reviewer:
+- karann
+- unniravindranathan
+
+---
+
+# Package Content
+
+It is possible to generate a URL to fetch an arbitrary package's content (the .nupkg file) using the V3 API. The
+resource used for fetching package content is the `PackageBaseAddress` resource found in the
+[service index](service-index.md). This resource also enables discovery all versions of a package, listed or unlisted.
+
+This resource is commonly referred to as either the "package base address" or as the "flat container".
+
+## Versioning
+
+There is one `@type` value for the package base address: `PackageBaseAddress/3.0.0`.
+
+## Base URL
+
+The base URL for the following APIs is the value of the `@id` property associated with the aforementioned
+resource `@type` value. In the following document, this placeholder URL will be used:
+
+```
+https://example/flatcontainer/
+```
+
+## HTTP methods
+
+All URLs found in the package base resource only support the HTTP method `GET`.
+
+## Enumerate package versions
+
+If the client knows a package ID and wants to discover which package versions the V3 package source has available, the
```

Comment:
What should be shown for SemVer 2 packages with build metadata? Include it? Exclude it?

Reply:
I'll clarify.

Build metadata should be excluded. The idea is that the versions in this list can be used verbatim as `VERSION` replacement values in subsequent URLs, e.g. the URL to the .nupkg.

Reply:
Fixed.